src/js/components/Meter/stories/Circle.js

```diff
@@ -1,37 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { storiesOf } from '@storybook/react';
 
 import { Grommet, Box, Meter } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-class CircleMeter extends React.Component {
-  state = { value: 20 };
+function CircleMeter() {
```

Comment:
I believe the default is to use `const` insead `function`.

Reply:
Agreed